README.md

```diff
@@ -29,51 +35,48 @@ WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
 
-You should have received a copy of the GNU General Public License
-along with Octave; see the file COPYING.  If not, see
-<https://www.gnu.org/licenses/>.
+You should have received a copy of the 
+[`GNU General Public License`](https://github.com/gnu-octave/octave/blob/default/COPYING)
+along with Octave.  If not, see
+[www.gnu.org/licenses](https://www.gnu.org/licenses).
 
-Availability
-------------
+### Availability
 
 The latest released version of Octave is always available from
-<https://ftp.gnu.org/gnu/octave/> and many mirror sites around the
-world.  You may also find links to binary distributions at
-<https://www.octave.org/download.html>.  The current development
-sources may be found under the Source Code tab on
-[Savannah](https://savannah.gnu.org/projects/octave/).
-
-Installation
-------------
+[here](https://ftp.gnu.org/gnu/octave)
+and many mirror sites around the world.  You may also find links to binary distributions
+[here](https://www.octave.org/download.html).
+The current development sources may be found under the Source Code tab on [Savannah](https://savannah.gnu.org/projects/octave/).
 
-Octave requires approximately 475 MB of disk storage to unpack and
-compile from source (significantly more, 3.8 GB, if you compile with
-debugging symbols).  Once installed, Octave requires approximately
-75 MB of disk space (again, considerably more, 415 MB, if you don't
-build shared libraries or the binaries and libraries include
-debugging symbols).
+## Developer Installation
 
 To compile Octave, you will need a recent version of:
 
 - [GNU Make](https://www.gnu.org/software/make/)
 - [GNU G++](https://gcc.gnu.org/) or another C++11 compiler
 - [GNU Fortran](https://gcc.gnu.org/fortran/), another Fortran 77
-  compiler, or [f2c](http://www.netlib.org/f2c/)
+  compiler, or [Fortran 77 to C (f2c)](http://www.netlib.org/f2c/)
 
 Octave's Makefiles use features of GNU Make that are not present in
 other versions of make.  If you use `f2c`, you will need a script
 like `fort77` that works like a normal Fortran compiler by combining
 `f2c` with your C compiler in a single script.
 
-See the notes in the file `INSTALL.OCTAVE` of the Octave source
-distribution for more detailed installation instructions.
+Octave requires approximately `475 MB` of disk storage to unpack and
+compile from source (significantly more, 3.8 GB, if you compile with
+debugging symbols).  Once installed, Octave requires approximately
+`75 MB` of disk space (again, considerably more, 415 MB, if you don't
+build shared libraries or the binaries and libraries include
+debugging symbols).
 
-Bugs and Patches
-----------------
+See [Octave installation guide](https://github.com/gnu-octave/octave/blob/default/doc/interpreter/install.txi)
```

Comment:
`.txi` files are sources from which documentation is generated. User shouldn't need to read those sources.

Maybe, use a link to the generated HTML documentation?
https://docs.octave.org/latest/Installation.html